app/assets/javascripts/discourse/app/components/user-status-picker.js

```diff
@@ -0,0 +1,50 @@
+import Component from "@ember/component";
+import { action, computed } from "@ember/object";
+import { scheduleOnce } from "@ember/runloop";
+import { emojiUnescape } from "discourse/lib/text";
+import { escapeExpression } from "discourse/lib/utilities";
+
+export default class UserStatusPicker extends Component {
```

Comment:
I would prefer if it was `tagName = null`, should be doable on this component

Reply:
I didn't know that we can get rid of wrapping div on classic components. But I noticed that `tagName = null` doesn't have any effect, after adding it the div is still there. But `tagName = ""` do the job (see [this answer](https://stackoverflow.com/a/23370774/3206146)).

Reply:
Yes sorry I meant `""`